readers/postgres/messages.go

```diff
@@ -89,6 +88,23 @@ func (tr postgresRepository) ReadAll(chanID string, offset, limit uint64, query
 	return page, nil
 }
 
+func fmtCondition(chanID string, query map[string]string) string {
+	condition := `channel = :channel`
+	for name := range query {
+		switch name {
+		case
+			"subtopic",
+			"publisher":
+			condition = fmt.Sprintf(`%s AND %s = :%s`, condition, name, name)
+		case
+			"name",
+			"protocol":
+			condition = fmt.Sprintf(`%s AND "%s" = :%s`, condition, name, name)
```

Comment:
What's the difference between these two sprintfs?

Reply:
One of them use `"` around the name, and one of them doesn't. I have copied them from InfluxDB, and I haven't change them, but I think we can merge them into one single Sprintf.

Reply:
Ah, yes indeed... It is too late here, I am tired :).

Reply:
If you can merge it in one sprintf and make it more readable - cool.

Otherwise - this already looks good to me.

Reply:
@1995parham Please check if this is possible, test it, and merge in one `Sprintf` is enough. If not, please let us know why not, I'm curious.

Reply:
Any progress on this?

Reply:
@drasko Sorry for the delay, I will report it until tonight.